Sources/SecureXPC/Server/ClientRequirement.swift

```diff
@@ -163,6 +170,34 @@ fileprivate protocol MessageAcceptor {
     func isEqual(to acceptor: MessageAcceptor) -> Bool
 }
 
+fileprivate struct HardenedMessageAcceptor: MessageAcceptor {
```

Comment:
Could you spaces instead of tabs for consistency? At least as shown by GitHub, the indentation here noticeably does not match the rest of this file.